nix/flake.nix

```diff
@@ -113,7 +119,9 @@
         '';
 
         postPatch = ''
-          echo ${version} > .tag
+          if [ ! -f .tag ]; then
+            echo ${version} > .tag
```

Comment:
it's important (for my sanity) that the version number correlates with the version number that wezterm's build would have generated for the same git revision. Please share examples of what is being injected here!

Reply:
this pr aligns the version number with that of normal releases, instead of using just the first 8 characters of commit hash without date and time information. for this particular change, it continues to use whatever version is defined to (in this PR `wezterm-00000000-000000-xxxxxxxx`), unless there is an existing tag file which is useful for cases when the source tar is downloaded and the version cannot be reliably retrieved.

these few lines are simply bash, not nix.